packages/conversations/__tests__/__dataSets__/get.ts

```diff
@@ -0,0 +1,532 @@
+import {
+  ConversationPageResponse,
+  ConversationPage,
+  Conversation,
+  ListConversationsParameters,
+  Member,
+  MemberPageResponse,
+  SessionPageResponse,
+  SessionPage,
+  EventPageResponse,
+} from '../../lib';
+
+import {
+  BASE_URL,
+  conversation,
+  conversationResponse,
+  member,
+  memberResponse,
+  session,
+  sessionResponse,
+  sessionUser,
+  event,
+  eventResponse,
+} from '../common';
+
+export default [
+  {
+    label: 'retrieve all conversations',
+    requests: [
+      [
+        `/v1/conversations`,
+        'GET',
+      ],
+      [
+        `/v1/conversations?cursor=next`,
+        'GET',
+      ],
+    ],
+    responses: [
+      [
+        200,
+        {
+          page_size: 10,
+          _embedded: {
+            conversations: [conversationResponse],
+          },
+          _links: {
+            self: {
+              href: '/v1/conversations',
+            },
+            next: {
+              href: 'https://api.nexmo.com/v1/conversations?cursor=next',
+            }
+          },
+          conversationResponse,
+        } as ConversationPageResponse,
+      ],
+      [
+        200,
+        {
+          page_size: 10,
+          _embedded: {
+            conversations: [{
+              ...conversationResponse,
+              id: 'CON-00000000-0000-0000-0000-000000000002',
+            }],
+          },
+          _links: {
+            self: {
+              href: '/v1/conversations',
+            },
+          },
+          conversationResponse,
+        } as ConversationPageResponse,
+      ],
+    ],
+    clientMethod: 'listAllConversations',
+    parameters: [
+    ],
+    generator: true,
+    error: false,
+    expected: [
+      conversation,
+      {
+        ...conversation,
+        id: 'CON-00000000-0000-0000-0000-000000000002',
+      } as Conversation
+    ],
+  },
+  {
+    label: 'list all conversations by user',
+    requests: [
+      [
+        `/v1/users/${member.user.id}/conversations`,
+        'GET',
+      ],
+      [
+        `/v1/users/${member.user.id}/conversations?cursor=next`,
+        'GET',
+      ],
+    ],
+    responses: [
+      [
+        200,
+        {
+          page_size: 10,
+          _embedded: {
+            conversations: [conversationResponse],
+          },
+          _links: {
+            self: {
+              href: '/v1/conversations',
```

Comment:
```suggestion
              href: `/v1/users/${member.user.id}/conversations`,
```
I think it's supposed to be this.